config/index.js

```diff
@@ -442,6 +442,19 @@ var conf = convict({
       format: RegExp,
       default: /.+@mozilla\.com$/,
       env: 'SIGNIN_CONFIRMATION_FORCE_EMAIL_REGEX'
+    },
+    bypassAccountWithAge: {
+      enabled :{
+        doc: 'Bypass sign-in confirmation for accounts with an age of at least X.',
+        default: false,
+        env: 'SIGNIN_CONFIRMATION_BYPASS_ACCOUNT_AGE_ENABLED'
+      },
+      accountCreatedSinceMS: {
+        doc: 'Account age in MS.',
+        format: 'duration',
+        default: '4 hours',
+        env: 'SIGNIN_CONFIRMATION_BYPASS_ACCOUNT_AGE_MS'
```

Comment:
Naming nit: the "MS" suffix here is inconsistent with other duration values in this file, and probably not necessary now that this takes a duration string rather than an integer.